QATCH/processors/Analyze.py

```
@@ -280,11 +280,11 @@ def Model_Data(data_path):
                     # new maths for resonance and dissipation (scaled)
                     avg = np.average(resonance_frequency[t_0p5:t_1p0])
                     ys = ys * avg / 2
-                    # ys_fit = ys_fit * avg / 2
+                    ys_fit = ys_fit * avg / 2
```

Ruff (0.15.11)

[error] Undefined name `ys_fit`

(F821)